MODULE.bazel.lock

```diff
@@ -155,6 +155,23 @@
   },
   "selectedYankedVersions": {},
   "moduleExtensions": {
+    "//rs/tests:kubeconfig_extension.bzl%kubeconfig_extension": {
+      "general": {
+        "bzlTransitiveDigest": "xEGrJHac1Q0AeSsq47wmCUw9/QuvIsSjn2H4zlg7RgM=",
+        "usagesDigest": "nx/HYUUCtKW9bl/h/wrK069HawIvbPlJo+fUzOBVDlQ=",
+        "recordedFileInputs": {},
+        "recordedDirentsInputs": {},
+        "envVariables": {},
+        "generatedRepoSpecs": {
+          "kubeconfig": {
+            "bzlFile": "@@//rs/tests:kubeconfig.bzl",
+            "ruleClassName": "kubeconfig_rule",
+            "attributes": {}
+          }
+        },
+        "recordedRepoMappingEntries": []
+      }
+    },
```

Comment:
@basvandijk the build IC test is was failing [constantly](https://github.com/dfinity/ic/actions/runs/11282190390/job/31379128254?pr=1877), until I added this to PR. It seems quite unrelated to my change, but tests are fixed now. Does this look safe to you?

Reply:
Yes, @nmattia just merged [a fix](https://github.com/dfinity/ic/commit/4a0c41566e410e17bb4d1e636d03a6f90088b505) for that and we'll be removing `MODULE.bazel.lock` for now until it stabilises: https://github.com/dfinity/ic/pull/1990.